components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -1285,6 +1285,7 @@ define([
                 } else {
                     if (label) {label.update("Linked to patient record");}
                     suggestContainer.hide();
+                    link.target = _this.targetNode.getPhenotipsPatientId();
```

Comment:
i think it should be `value` not `_this.targetNode.getPhenotipsPatientId()`. After all this is what is used to generate the url on the very next line